exoboot.py

```diff
@@ -312,7 +320,10 @@ def read_data(self, loop_time=None):
             self.data.toe_fsr = self.toe_fsr_detector.value
         if self.do_include_sync:
             self.data.sync = self.sync_detector.value
-
+        self.data.rise = config.RISE_FRACTION
```

Comment:
what happens if the controller is not 4 point spline?

Reply:
it will log the default value